src/analyser.h

```diff
@@ -14,14 +14,15 @@
 #include "trackinfoobject.h"
 
 class Analyser {
-
-public:
-    virtual bool initialise(TrackPointer tio, int sampleRate, int totalSamples) = 0;
+  public:
+    virtual bool initialise(TrackPointer tio, int sampleRate,
+                            int totalSamples) = 0;
```

Comment:
These type of changes do not help for readability. Is there a way to specify a soft 80 columns limit? 
The hard limit should be at 100 or so. Github allows 115.